filters/apiusagemonitoring/spec.go

```diff
@@ -96,11 +97,17 @@ func NewApiUsageMonitoring(
 
 // apiConfig is the structure used to parse the parameters of the filter.
 type apiConfig struct {
-	ApplicationId         string   `json:"application_id"`
-	Tag                   string   `json:"tag"`
-	ApiId                 string   `json:"api_id"`
-	PathTemplates         []string `json:"path_templates"`
-	ClientTrackingPattern string   `json:"client_tracking_pattern"`
+	ApplicationId         string           `json:"application_id"`
+	Tag                   string           `json:"tag"`
+	ApiId                 string           `json:"api_id"`
+	PathTemplates         []string         `json:"path_templates"`
+	ClientTrackingPattern string           `json:"client_tracking_pattern"`
+	Context               apiConfigContext `json:"context"`
```

Comment:
context may be not the right name for this, just don't have a good suggestion at the moment